src/orange/orangeinp/Solid.hh

```diff
@@ -0,0 +1,185 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/orangeinp/Solid.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <optional>
+#include <type_traits>
+#include <utility>
+
+#include "corecel/math/Turn.hh"
+
+#include "ConvexRegion.hh"
+#include "ObjectInterface.hh"
+
+namespace celeritas
+{
+namespace orangeinp
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Define the angular region of a solid.
+ *
+ * This angle is a cross section perpendicular to the z axis, with a start
```

Comment:
The angle is not a cross section perpendicular to the z axis, but rather it is defined on a cross section perpendicular to the z axis.